subjects/rockandroll/README.md

```diff
@@ -0,0 +1,45 @@
+## rockandroll
+
+### Instructions
+
+Write a function called `RockAndRoll` that takes an `int` and returns a `string`.
+
+- If the number is divisible by 2, print `rock` followed by a newline `\n`.
+- If the number is divisible by 3, print `roll` followed by a newline `\n`.
+- If the number is divisible by 2 and 3, print `rock and roll` followed by a newline `\n`.
+
```

Comment:
Missing two edge cases (maybe three depending on whether we should test for overflows or not). What happens with negative numbers, and numbers that are not divisible by 2 or 3). Function prints the number on these two cases

Reply:
I think it is enough that the test already covers these cases There is no need to be present in the subject.

Reply:
I think it is enough, the test already covers these cases. There is no need to be present in the subject.